docs/content/concepts/graphql-rpc.mdx

```diff
@@ -363,17 +366,13 @@ query {
 ```graphql
 query {
   address(address: "0x5094652429957619e6efa79a404a6714d1126e63f551f4b6c7fb76440f8118c9") {
-    coinConnection(
+    coins(
       first: 3, 
-      after: "0x5094652429957619e6efa79a404a6714d1126e63f551f4b6c7fb76440f8118c9", 
+      after: "IAB3ha2PEA4ESRF4UErsJufJEwYpmSbCq7UNpxIHnLhG", # note this is Base64 encoded
```

Comment:
I wouldn't go to too much trouble explaining that it's Base64 encoded -- this value is supposed to be opaque, so it's counter to purpose to explain how we're hiding the data.